libbuteosyncfw/profile/ProfileManager.cpp

```diff
@@ -1137,7 +1149,8 @@ QDomDocument ProfileManagerPrivate::constructProfileDocument(const Profile &aPro
 bool ProfileManagerPrivate::writeProfileFile(const QString &aProfilePath,
         const QDomDocument &aDoc)
 {
-    //FUNCTION_CALL_TRACE;
+    FUNCTION_CALL_TRACE;
+    LOG_WARNING("writeProfileFile() called, forcing disk write:" << aProfilePath);
```

Comment:
http://doc.qt.io/qt-5/qiodevice.html#OpenModeFlag-enum "QIODevice::WriteOnly	0x0002	The device is open for writing. Note that this mode implies Truncate."